src/lib/Select/Select.svelte

```diff
@@ -139,6 +141,11 @@
       role="button"
       tabindex="0"
     >
+    {#if properties.icon && properties.iconPath}
```

Comment:
add empty string check or default to null above

Reply:
done